packages/compiler/src/compiler/foo.test.ts

```diff
@@ -0,0 +1,30 @@
+import { describe, expect, it } from 'vitest'
+
+import { Compiler } from './compile'
+
+describe('Compiler', () => {
+  it('test', async () => {
+    const prompt = `
+---
+model: chat-bison-001
+temperature: 0.9
+tools:
+  - number1
+  - number2
+---
+
+Erase una vez un señor llamado {{name}}.
+`.trim()
+
+    const compiler = new Compiler({
+      prompt,
+      parameters: {
+        name: 'Latitude',
+      },
+    })
+
+    const conversation = await compiler.compile()
+
+    console.log(JSON.stringify(conversation, null, 2))
+  })
+})
```

Comment:
lol the filename, maybe something more meaningful?

Reply:
oh I pushed it by accident xd, was just testing some things. Will do a proper test when finished.